tests/test_wsgi_apps.py

```diff
@@ -29,4 +29,4 @@ def test_default(app: SphinxTestApp, status: StringIO, warning: StringIO,
 
   img_path = app.outdir / imgs[0]['src']
   with open(img_path, "rb") as fd:
-    data_regression.check(fd.read())
+    image_regression.check(fd.read(), diff_threshold=0)
```

Comment:
```
FAILED tests/test_wsgi_apps.py::test_default - AssertionError: Difference between images too high: 0.019000842524509804 %
```

I believe the CI error comes from the subtle rendering difference between browsers / OSes. Can we relax this threshold to accept the insignificant difference?